### PR TITLE
MMB: Add boarding pass return

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  +data: BoardingPassReturnType,
+|};
+
+export const BoardingPassReturn = (props: Props) => {
+  const outbound = idx(props.data, _ => _.outbound.legs) || [];
+  return (
+    <React.Fragment>
+      <View style={styles.row}>
+        <TextIcon code="&#xe079;" style={styles.outboundIcon} />
+        <Translation passThrough="departure" />
```
Comment (RobinCsl) [inline app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js]: This is temporary, right?

File: app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+export default createFragmentContainer(
+  BoardingPassReturn,
+  graphql`
+    fragment BoardingPassReturn on BookingReturn {
+      outbound {
+        legs {
+          id
+          departure {
+            localTime
```
Comment (RobinCsl) [inline app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js]: Is this field necessary in the query?

File: app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js (hunk 1 above)
Comment (tbergq) [reply app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js]: yes, just trying to keep the PR's small

File: app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js (hunk 2 above)
Comment (tbergq) [reply app/MMB/src/scenes/tickets/boardingPasses/BoardingPassReturn.js]: probably not yet, but in my next PR I will show date and time.

**Comment by tbergq** [comment]
@RobinCsl removed `localTime`

**Pull request merged**